lhotse/utils.py

```diff
@@ -609,6 +609,10 @@ def is_none_or_gt(value, threshold) -> bool:
     return value is None or value > threshold
 
 
+def is_equal_or_contains(value, other) -> bool:
+    return value == other or (isinstance(value, Iterable) and other in value)
```

Comment:
I'm sure Iterable is not the right type here, my best guess is Sequence, but you should check the documentation of `typing` to make sure.

```suggestion
    return value == other or (isinstance(value, Sequence) and other in value)
```

Reply:
would be nice to add types e.g. `value: Any, other: Union[Any, Sequence[Any]]` (or if you feel adventurous, add proper type vars to indicate preserving types across arguments, see TypeVar in `typing` IIRC)